conf/config_test.go

```diff
@@ -105,31 +105,72 @@ func TestUnsetEnv(t *testing.T) {
 	}
 }
 
-func Test_setup(t *testing.T) {
-	type args struct {
-		dir string
-	}
+func TestLoadConfigs(t *testing.T) {
 	tests := []struct {
 		name string
-		args args
+		args []string
 		want string
 	}{
 		{
 			"See if Viper sets up correctly",
-			args{os.Getenv("GOPATH") + "/src/github.com/CMSgov/bcda-app/conf/test/local.env"},
+			[]string{os.Getenv("GOPATH") + "/src/github.com/CMSgov/bcda-app/conf/test/local.env"},
 			"true",
 		},
+		{
+			"test multiple locations for config files",
+			[]string{os.Getenv("GOPATH") + "/src/github.com/CMSgov/bcda-app/conf/test/local.env", os.Getenv("GOPATH") + "/src/github.com/CMSgov/bcda-app/conf/test/notlocal.env"},
+			"false",
+		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			v, state := setup(tt.args.dir)
+			v, state := loadConfigs(tt.args...)
 			if !reflect.DeepEqual(v.Get("TEST").(string), tt.want) {
 				t.Errorf("setup() = %v, want %v", state, tt.want)
 			}
 		})
 	}
 }
 
+func TestGetConfigPaths(t *testing.T) {
```

Comment:
this test might be overkill, but didn't want a drop in coverage after small init() refactor.